Formula/rsdns.rb

```diff
@@ -0,0 +1,48 @@
+class Rsdns < Formula
+  desc "Rackspace Cloud DNS CLI"
+  homepage "https://github.com/linickx/rsdns"
+  url "https://github.com/linickx/rsdns/archive/v4.0.tar.gz"
+  sha256 "060bea37a85c9debe59bb23c8a4326e97ced04fe3de15689e34123c978dffb99"
+  head "https://github.com/linickx/rsdns.git"
+
+  bottle :unneeded
+
+  depends_on "jq"
+
+  # This package is essentially a web of shell script that all expect their
+  # friends to be in some particular location. Fixing that upstream to be more
+  # homebrew friendly is a lot of work compared to the simple patch that can
+  # workaround this.
+  patch :DATA
```

Comment:
As requested before: can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. 

Alternatively using a wrapper script or symlink may be another option. Thanks!